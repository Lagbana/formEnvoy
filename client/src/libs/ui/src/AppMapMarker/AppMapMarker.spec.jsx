import React from 'react'
import { shallow, configure } from 'enzyme'
import toJson from 'enzyme-to-json'
import Adapter from 'enzyme-adapter-react-16'
import { AppMapMarker } from '.'

configure({ adapter: new Adapter() })

test('<AppMapMarker /> renders correctly', () => {
  const component = shallow(
    <AppMapMarker coordinate={{ latitude: 0, longitude: 0 }} />
  )
  expect(toJson(component)).toMatchSnapshot()
})
