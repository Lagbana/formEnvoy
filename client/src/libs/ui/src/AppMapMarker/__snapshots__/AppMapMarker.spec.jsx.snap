// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppMapMarker /> renders correctly 1`] = `
<AIRMapMarker
  coordinate={
    Object {
      "latitude": 0,
      "longitude": 0,
    }
  }
  onPress={[Function]}
  stopPropagation={false}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
        "position": "absolute",
      },
      undefined,
    ]
  }
/>
`;
